geode-core/src/main/java/org/apache/geode/cache/client/internal/ExecuteRegionFunctionOp.java

```diff
@@ -310,29 +198,22 @@ public static void reexecute(ExecutablePool pool, String region, String function
 
     private FunctionException functionException;
 
-
-    public ExecuteRegionFunctionOpImpl(String region, Function function,
-        ServerRegionFunctionExecutor serverRegionExecutor, ResultCollector rc, byte hasResult,
+    private static int getMsgParts(int filterSize,
```

Comment:
getMsgPartCount would be a bit clearer

Reply:
I think getMessagePartCount is even more clear. Acronyms and abbreviations should really be avoided. The IDE saves us from having to write longer method names and in reality code becomes more readable with clearer method names

Reply:
I agree. I will leave getMessagePartCount.